prompts/precision_v2.txt

```diff
@@ -0,0 +1,94 @@
+prompts/precision_v2.txt
+A list of records will be provided from an ontology of disease terms. Each record will contain information describing a single term.
+
+Assign a `precision` label to each of these terms that captures the extent to which they correspond to patient populations with distinguishing clinical, demographic, physiological or molecular characteristics. Use exactly one of the following values for this label:
+
+- `high`: High precision terms have the greatest ontological specificity, sometimes (but not necessarily) correspond to small groups of relatively homogeneous patients, often have greater diagnostic certainty and typically represent the forefront of clinical practice.
+- `medium`: Medium precision terms are the ontological ancestors of `high` precision terms (if any are known), often include indications in later stage clinical trials and generally reflect groups of patients assumed to be suffering from a condition with a shared, or at least similar, physiological or environmental origin.
+- `low`: Low precision terms are the ontological ancestors of both `medium` and `high` precision terms, group collections of diseases with *some* shared characteristics and typically connote a relatively heterogenous patient population. They are often terms used within the ontology for organizational purposes.
+
+The records provided will already have the following fields:
+
+- `id`: A string identifier for the term
+- `label`: A descriptive name for the term
+- `description`: A longer, possibly truncated description of what the term is; may be NA (i.e. absent)
+
+Here is a list of such records (in YAML format) where the `precision` label is already assigned for 3 examples at each level of precision:
+
+--- BEGIN EXAMPLES ---
```

Comment:
Minor preference for sorting these by precision label so all the lows are together, then the mediums, etc.

Reply:
Cool, will make the change